lib/developer_portal/lib/liquid/forms/account.rb

```diff
@@ -8,7 +8,17 @@ def html_class_name
         end
 
         def path
-          admin_account_payment_details_path
+          if payment_gateway_type == :stripe
+            admin_account_stripe_path
```

Comment:
This is in order for `update` method to be executed on `StripeController`, and not the generic `PaymentDetailsBaseController`, so that billing address is updated on the payment method.

Reply:
TODO: `admin_account_#{payment_gateway_type}_path` (smth like this)